### PR TITLE
Site upgrade

### PR DESCRIPTION
**Pull request by andriiivaniuk**
Hover effects, animations, fixes, hamburger button for header menu from 500px wide, video.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -190,14 +193,14 @@ <h2 class = "section__header">
...
+                                    <span class = "location-in-focus__text-element__info">
                                         Jump off balcony, onto stranger's head. Chase ball of string
                                         hide when guests come over. Being gorgeous with belly side
                                         up i could pee on this if i had the energy but under the bed,
                                         for attack the child, open the door, 
-                                    </div>
-                                    <a href="">View Neighborhood &#160 &#160 &#160
-                                        <img src="images/arrow.svg" alt="arrow">
+                                    </span>
+                                    <a class="blue-link-with-arrow" href="">
```
Comment (vladhybalo) [inline index.html:200]: image
Could you pls adapt it for mobile?

File: index.html [hunk 2]
```diff
@@ -208,7 +211,8 @@ <h2 class = "section__header">
                     </section>
                     <article>
                         <div class = "today-places-to-visit" id="today-places-to-visit">
-                            <div class = "section-inspire__header">Today top places to visit
+                            <div class = "section-inspire__header">
```
Comment (vladhybalo) [inline index.html]: Why is it still `div`?

**Comment by vladhybalo** [comment]
image
Looks like there is small horizontal scrolling, could you pls get red of it

**Comment by vladhybalo** [comment]
hamburger doesnt move back after menu list hiding
image

File: css/style.css [hunk 3]
```diff
@@ -14,19 +22,86 @@ body{
...
 }
 
+#toggle-input{
+    display: none;
+}
+
+@media only screen and (max-width: 499px){
+    .header__top_items > .header__top_items_link{
+        display: none;
+    }
```
Comment (vladhybalo) [inline css/style.css]: as we are trying to implement by BEM, pls get rid of such selectors.
PS. maybe you should move your media to the bottom of the file?

File: css/style.css [hunk 4]
```diff
@@ -1,3 +1,11 @@
+:root{
+    --color-light-blue: #006DFE;
+    --footer-color: #0E1D28;
+    --button-overlay-color: rgb(136, 11, 22);
+    --header-image-overlay-color: rgba(16, 30, 59, 0.5);
+    --animation-speed: 0.23s;
+}
+
```
Comment (vladhybalo) [inline css/style.css:8]: image
These circles should be pseudoelements.

File: css/style.css [hunk 5]
```diff
@@ -122,15 +208,15 @@ body{
         margin-top: 150px;
     }
 
+    .combine_emelent_with_circle{
```
Comment (vladhybalo) [inline css/style.css]: Interesting things with class naming) pls check again BEM approach and try to implement it)

File: css/style.css [hunk 6]
```diff
@@ -189,32 +304,44 @@ body{
...
+
+.header__top_items_link::after{
+    position: absolute;
+    content: '';
+    bottom: -3px;
+    left: 0;
+    height: 2px;
+    background-color: whitesmoke;
+    width: 0%;
+    transition: var(--animation-speed) ease-in-out;
```
Comment (vladhybalo) [inline css/style.css]: It's a good practice to specify properties that you are going to animate.

File: css/style.css [hunk 7]
```diff
@@ -280,7 +408,7 @@ header{
 .combine_emelent_with_circle{
     display: flex;
     align-items: center;
-    padding-right: 20px;
+    /* padding-right: 20px; */
```
Comment (vladhybalo) [inline css/style.css]: Could you pls remove all unused code)

File: css/style.css [hunk 8]
```diff
@@ -529,6 +748,23 @@ header{
...
+    content: '';
+    height: 3px;
+    width: 0%;
+    background-color: var(--color-light-blue);
+    margin-top: 7px;
+    transition: var(--animation-speed) ease-in-out;
+}
+
+.location-in-focus__text-element__header:hover::after
+{
```
Comment (vladhybalo) [inline css/style.css]: Looks like these styles are copy-paste, could you pls move them to some reusable class, and reuse it

File: css/style.css [hunk 9]
```diff
@@ -690,15 +944,23 @@ header{
...
 }
 
-.footer__set__logo-element > img{
+.footer__set__logo-element__logo-wrapper{
+    flex-grow: 1;
     max-width: 170px;
     max-height: 65px;
 }
 
+.footer__set__logo-element__logo-wrapper img{
```
Comment (vladhybalo) [inline css/style.css]: bem

File: css/style.css [hunk 10]
```diff
@@ -135,17 +160,17 @@ body{
         gap: 5px;
```
Comment (nastiaymlnv) [inline css/style.css:170]: It's recommended to replace gap with other properties like margin or padding

**Review by nastiaymlnv — APPROVED** [review]
Good job!

**Pull request merged**